processing/src/main/java/org/apache/druid/java/util/metrics/CgroupCpuMonitor.java

```diff
@@ -67,7 +71,16 @@ public boolean doMonitor(ServiceEmitter emitter)
         "cgroup/cpu/cores_quota",
         computeProcessorQuota(cpuSnapshot.getQuotaUs(), cpuSnapshot.getPeriodUs())
     ));
+    emitter.emit(builder.setMetric("cgroup/cpuacct/usage", cpuSnapshot.getUsageNs()));
```

Comment:
I think I messed up by not putting all of these metric names into a single constant file `CgroupUtil`. In a subsequent PR, do you think you can do that consolidation work?

Reply:
sure.